nodejs/listeners/views/newTaskModal.js

```diff
@@ -58,12 +59,20 @@ module.exports = (app) => {
           slackWorkspaceID: body.team.id,
         },
         include: [
-          Task,
+          {
+            model: Task,
+            as: 'createdTasks',
+          },
+          {
+            model: Task,
+            as: 'assignedTasks',
+          },
         ],
       });
       const user = queryResult[0];
 
-      const storedTask = await user.createTask(task);
+      console.log(task);
+      const storedTask = await user.createCreatedTask(task);
```

Comment:
We still need to figure out this section, as the database expects a reference to the Users table, not a slack user ID

Reply:
It's also really weirdly written. I might see if I can make it read less weird.